lib/persist.php

```diff
@@ -107,6 +108,6 @@ function sanitizeChartConfig($charts) {
   array("data" => $data),
   array("response" => $response)
 ));
-REDCap::logEvent("Vizr chart configuration changed", $logData, NULL/*sql*/,
+\REDCap::logEvent("Vizr chart configuration changed", $logData, NULL/*sql*/,
```

Comment:
I wonder if we should use [module logging](https://github.com/vanderbilt-redcap/external-module-framework-docs/blob/main/methods/logs.md), now that it's availble.

Reply:
So something like?
```
- \REDCap::logEvent("Vizr chart configuration changed", $logData, NULL/*sql*/,
+ $module->log("Vizr chart configuration changed", 
+     [
+         "config_data" => $data, 
+         "response" => $response,
+         "PID" => $project_id
+     ]
+ );
```

Reply:
That looks reasonable, but I've never used it before. I'd suggest trying different things locally and seeing what shows up in the external module logs interface. The link is in the Control Center, next to the link to manage EMs.